### ISSUE TITLE
Allow for subtracting buffer heats

### ISSUE THREAD

**Issue by bas-rustenburg** [choderalab/bayesian-itc#74]
We need to add a feature to subtract buffer heats from the experiment to correctly run the MCMC script.

**Comment by jchodera** [comment]
Can you clarify?

**Comment by jchodera** [comment]
The Bayesian ITC scheme uses an explicit model for the buffer mechanical heats rather than fitting the difference heats.

**Comment by bas-rustenburg** [comment]
Yes, that is one constant heat. However, it might be useful to add the possibility to subtract the heat from a ligand-buffer titration, where a dilution effect may occur.
